wagtail/admin/views/reports/page_types_usage.py

```diff
@@ -0,0 +1,175 @@
+import django_filters
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.exceptions import PermissionDenied, ValidationError
+from django.db.models import Count, F, OuterRef, Q, Subquery
+from django.utils.translation import gettext_lazy as _
+
+from wagtail.admin.filters import WagtailFilterSet
+from wagtail.admin.views.reports import ReportView
+from wagtail.coreutils import get_content_languages
+from wagtail.models import ContentType, Page, Site, get_page_models
+from wagtail.permission_policies.pages import PagePermissionPolicy
+from wagtail.users.utils import get_deleted_user_display_name
+
+
+def _get_locale_choices():
+    choices = [  # noqa: C416
+        (language_code, display_name)
+        for language_code, display_name in get_content_languages().items()
+    ]
+    return choices
+
+
+def _get_site_choices():
+    """Tuples of (site root page path, site display name) for all sites in project."""
+    choices = [(site.root_page.path, str(site)) for site in Site.objects.all()]
```

Comment:
Little optimisation opportunity here...
```suggestion
    choices = [(site.root_page.path, str(site)) for site in Site.objects.select_related("root_page").all()]
```